service_container/tags.rst

```diff
@@ -570,47 +570,113 @@ application handlers::
         }
     }
 
-.. tip::
+Prioritizing Tagged Services
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
We need a `.. versionadded:: 4.4` here, see https://symfony.com/doc/current/contributing/documentation/format.html#new-features-behavior-changes-or-deprecations.

Reply:
Done

Reply:
Added it. Thanks :)

Reply:
Missing a dot :)

Reply:
Added